cmd/gds/main.go

```diff
@@ -983,7 +983,7 @@ func adminStatus(c *cli.Context) (err error) {
 	defer cancel()
 
 	var rep *admin.StatusReply
-	if rep, err = adminClient.Status(ctx); err != nil {
+	if rep, err = adminClient.AdminStatus(ctx); err != nil {
```

Comment:
Does this mean we have some general name spacing issues? Is it just related to the embedding or something else?

Reply:
It's embedding related, in the unified `GlobalDirectoryClient` interface we can't have methods with the same name.

Reply:
Ok, since we're going to make the admin client separate from the GDS client, could we please change this back?